zubhub_frontend/zubhub/src/views/create_team/step1/Step1.jsx

```diff
@@ -49,11 +16,11 @@ export default function Step1({ formik, formValues, setFormValues, handleBlur })
             variant="outlined"
             name="groupname"
             placeholder="Choose a name that best suites your team"
```

Comment:
this also wasn't done by you but since you worked on this file can you translate it too?